src/tools/sssctl/sssctl_logs.c

```diff
@@ -50,6 +52,15 @@
     } \
 } while(0)
 
+#define POPT_SERV_OPTION(NAME) {services[SERV_ ## NAME].name, \
+                                '\0', POPT_BIT_SET, &pc_services, \
```

Comment:
Better pass pc_services as macro variable, instead of hardcoding it in macro itself.

Reply:
Sure!